# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/base/crashtests/1890888.html
# Commit: a6c95ed12e51
# Full Hash: a6c95ed12e514ccf3e43f7cf11dfa317ab376543
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2024-04-11 21:36:48
# Description:
#   Bug 1890888 - Fix a crash where mCrossShadowBoundaryRange is not clear when the shadow host is removed r=smaug
#   
#   If the end point is at a shadow root, and the corresponding host node is
#   removed from DOM, we should clear this mCrossShadowBoundaryRange for it.
#   
# ==============================================================================

diff -r 86827dc0b95b -r a6c95ed12e51 dom/base/crashtests/1890888.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/base/crashtests/1890888.html	Thu Apr 11 16:21:19 2024 +0000
@@ -0,0 +1,13 @@
+<script>
+window.addEventListener("DOMContentLoaded", () => {
+  o1.prepend(o4)
+  document.getSelection().setBaseAndExtent(o5.attachShadow({mode: "closed"}), 0, o3, 0)
+  o5.replaceWith(o2)
+  setTimeout(window.close, 500)
+})
+</script>
+<iframe id="o1"></iframe>
+<picture id="o2"></picture>
+<area id="o3">
+<meter id="o4">
+<p id="o5">